### PR TITLE
docs: quickstart guide

### PR DESCRIPTION
**Pull request by shanewxy**

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -33,61 +33,55 @@ For more detailed installation requirements, please refer to [installation](/dep
...
-![qs-create-svc](/img/v0.4.0/quickstart/qs-create-svc-en.png)
+![qs-create-res-nginx](/img/v0.5.0/quickstart/qs-create-res-nginx-en.png)
 
-3. Fill in the service name, such as `myapp`, and make sure the template is selected as `kubernetes-containerservice`.
-4. Enter `nginx` image in the Image Name of the template configuration, and click confirm.
+5. Click the `Save and Deploy` button to complete the creation of the resource.
+6. After the resource deployment is complete, you can view components, logs, execute terminal commands, access the address of this nginx service, etc.
 
-![qs-create-svc2](/img/v0.4.0/quickstart/qs-create-svc2-en.png)
+- Access the endpoint.
```
Comment (gitlawr) [inline docs/quickstart.md:71]: Add `-p 30000-30100:30000-30100` to the install command to make it work. Also, add a note to explain it.

File: docs/quickstart.md [hunk 2]
```diff
@@ -33,61 +33,55 @@ For more detailed installation requirements, please refer to [installation](/dep
...
-2. Click the `Connectors` tab.
-3. Click the `New Connector` button, and select `Kubernetes` type.
-4. Fill in the cluster name and KubeConfig file, and click save.
+Walrus provides a default `local` environment under the `default` project for you to quickly experience the application deployment process. 
 
-![qs-add-connector](/img/v0.4.0/quickstart/qs-add-connector-en.png)
+![qs-local-env](/img/v0.5.0/quickstart/qs-local-env-en.png)
 
-## Create Environment
+The `local` environment connects to a local Kubernetes cluster connector created by built-in K3s. You can use it to deploy applications without having to configure a Kubernetes cluster.
```
Comment (gitlawr) [inline docs/quickstart.md:48]: Add a note to clarify that the embedded k3s is not meant for production usage

File: i18n/zh/docusaurus-plugin-content-docs/current/quickstart.md [hunk 3]
```diff
@@ -37,7 +41,11 @@ sudo docker run -d --privileged --restart=always \
...
 ```
 
-![qs-first-login](/img/v0.4.0/quickstart/qs-first-login.png)
+![qs-first-login](/img/v0.5.0/quickstart/qs-first-login.png)
+
+3. 登陆后，您可以设置新密码，以及配置 Walrus 的访问地址。
+
+![qs-set-pwd](/img/v0.5.0/quickstart/qs-set-pwd.png)
 
 ## 模板加速
```
Comment (gitlawr) [inline i18n/zh/docusaurus-plugin-content-docs/current/quickstart.md:50]: 模板加速这里再写一段docker run感觉比较累赘，建议放到`注意`中一同解释-e SERVER_BUILTIN_CATALOG_PROVIDER=gitee参数的作用

Comment (shanewxy) [reply i18n/zh/docusaurus-plugin-content-docs/current/quickstart.md]: updated

File: docs/quickstart.md [hunk 4]
```diff
@@ -33,61 +37,57 @@ For more detailed installation requirements, please refer to [installation](/dep
...
+![qs-env-list](/img/v0.5.0/quickstart/qs-env-list-en.png)
 
-1. In the environment list page, click the environment name created in the previous step to enter the environment view.
+2. Click the `Import YAML` button to import the built-in Walrus File resource.
 
-![qs-go2env](/img/v0.4.0/quickstart/qs-go2env-en.png)
+![qs-create-res](/img/v0.5.0/quickstart/qs-create-res-en.png)
 
-2. Click the `New` button and select `Service`.
+3. Select the `nginx` resource from the Walrus File Hub resource list on the right and import the YAML.
```
Comment (gitlawr) [inline docs/quickstart.md:70]: Select the `nginx` example from the Walrus File Hub on the right.
To avoid conflicting with Walrus resource

Comment (shanewxy) [reply docs/quickstart.md]: updated

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**